test/libs/data_availability_report_test.py

```diff
@@ -0,0 +1,28 @@
+import io
+import pandas as pd
+from libs.datasets.latest_values_dataset import LatestValuesDataset
+from libs.qa import data_availability
+
+
+def test_build_availability_report():
+
+    input_csv = [
+        "fips,state,aggregate_level,population,field1,field2",
+        "36061,NY,county,500,1,",
+        "36062,NY,county,501,0,10",
+        "36063,NY,county,501,0,15",
+        "36,NY,state,10000,,10",
+        "09,CT,state,10000,10,10",
+    ]
+
+    dataset = LatestValuesDataset.load_csv(io.StringIO("\n".join(input_csv)))
```

Comment:
I like doing a "\n".join before calling read_csv. Maybe modify existing dataset_utils_test.read_csv_and_index_fips or add a method similar to it.